### PR TITLE
Cu 36n3c66 change loader from gif to css

### PR DESCRIPTION
**Pull request by amnong**
[Task](https://app.clickup.com/t/36n3c66).

See 2 gif videos in [this comment](https://app.clickup.com/t/36n3c66?comment=1140651594).

### CODE REVIEW

File: package.json
Comment (talkor) [inline package.json:9]: Why it's needed?

File: src/components/Action/Action.vue
Comment (talkor) [inline src/components/Action/Action.vue:80]: Did you change prettier settings? It's removing trailing commas 🙄

File: src/components/Loader/Loader.vue
Comment (talkor) [inline src/components/Loader/Loader.vue:60]: Avoid using elements as selector as they might change, use classes instead

Comment (talkor) [reply src/components/Loader/Loader.vue:60]: Also, we don't support postcss-nested here (yet)

File: src/components/Loader/Loader.vue
Comment (talkor) [inline src/components/Loader/Loader.vue:3]:
```suggestion
  <div v-if="secondary" class="loader-secondary" :class="{ hide }"></div>
  <div v-else class="spinner-container" :class="{ hide }">
```

File: src/components/Loader/Loader.vue
Comment (talkor) [inline src/components/Loader/Loader.vue:19]: I don't understand it's purpose

**Review by talkor — COMMENTED** [review]
TBH I like it less than the existing loader, it looks like a hot spot or a "call to tap" rather than something that indicated loading, I think we need UX's opinion on this first.
Also, please add some examples in Vuepress

File: package.json
Comment (amnong) [reply package.json:9]: I wanted changes in `@swimm/ui` to be reflected when working in the webapp. To make it work I had to:
1. In the webapp repo, run `yarn add link:$PATH_TO_SWIMM_UI` and then `yarn install`.
2. Add a `watch` script so code changes will trigger a build.

Did I miss an existing way to do this?

File: src/components/Action/Action.vue
Comment (amnong) [reply src/components/Action/Action.vue:80]: I was wondering about this myself actually. I didn't do anything related to prettier... 🤔
In my vscode settings I have this:
```json
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
```

And I also turned on format-on-save. For Swimm webapp it works fine 🤷‍♂️ 

According to our configuration, should we have trailing commas? How can I determine whether or not this is a problem?

File: src/components/Loader/Loader.vue
Comment (amnong) [reply src/components/Loader/Loader.vue:60]: Oh nice catch then, because in dev mode it works fine. Thanks!

File: src/components/Loader/Loader.vue
Comment (amnong) [reply src/components/Loader/Loader.vue:19]: This was here before my PR. No idea why 🤷‍♂️

**Comment by amnong** [comment]
@talkor thanks for reviewing :) I pushed a fix commit.

**Comment by amnong** [comment]
Oh I didn't see your comment about Vuepress - I'll do that tomorrow.

**Comment by amnong** [comment]
@yhamry could you please help reviewing this? I wanted to continue with Talko but he's swamped and cannot get to it. Will you be able to help?

File: src/components/Loader/Loader.vue
Comment (yhamry) [reply src/components/Loader/Loader.vue:34]: This discussion with the UX doesn't approve it - I would say @shirly should approve it before this is approved.

**Review by yhamry — CHANGES_REQUESTED** [review]
Waiting on UX approval - other than that - looks fine

**Review by yhamry — APPROVED** [review]

**Pull request merged**
